cmd/build.go

```diff
@@ -33,6 +33,9 @@ func BuildCmd(fs afero.Fs) *cobra.Command {
 				return fmt.Errorf("owner flag malformed: %w", err)
 			}
 
+			repo = GetRepo(repo)
```

Comment:
This function is not needed. The same functionality is provided by `viper` itself.

You should look at how using flags with viper: https://github.com/spf13/viper#working-with-flags

The default value should be moved from the flag to the `settings` package and set using `viper.SetDefault()`